libs/blocks/merch-cards/merch-cards.js

```diff
@@ -0,0 +1,257 @@
+import '../../deps/merch-cards.js';
+import '../../deps/merch-card.js';
+import '../../deps/commerce.js';
+import '../merch/merch.js';
+import '../merch-card/merch-card.js';
+import { createTag, decorateLinks, getConfig, loadBlock, loadStyle } from '../../utils/utils.js';
+import { replaceText } from '../../features/placeholders.js';
+
+const DIGITS_ONLY = /^\d+$/;
+
+const LITERAL_SLOTS = [
+  'searchText',
+  'filtersText',
+  'sortText',
+  'popularityText',
+  'alphabeticallyText',
+  'noResultText',
+  'resultText',
+  'resultsText',
+  'searchResultText',
+  'searchResultsText',
+  'searchResultMobileText',
+  'searchResultsMobileText',
+  'noSearchResultsText',
+  'noSearchResultsMobileText',
+  'showMoreText',
+];
+
+// allows improve TBT by returning control to the main thread.
+// eslint-disable-next-line no-promise-executor-return
+const makePause = async () => new Promise((resolve) => setTimeout(resolve, 0));
+
+const fail = (el, err = '') => {
+  window.lana?.log(`Failed to initialize merch cards: ${err}`);
+  el.innerHTML = '';
+  return el;
+};
+
+/**
+ * Removes merch cards from the DOM if they are not meant to be displayed in this merch cards block.
+ * @param {*} merchCards merch-cards element
+ */
+export function filterMerchCards(merchCards, filtered) {
+  [...merchCards.children].filter((child) => child.tagName === 'MERCH-CARD')
+    .forEach((card) => {
+      if (!Object.prototype.hasOwnProperty.call(card.filters, filtered)) {
+        card.remove();
+      }
+    });
+}
+
+/** parse card preferences for each category filter */
+export function parsePreferences(elements) {
+  return [...elements].map((el) => {
+    let size;
+    if (el?.firstElementChild?.tagName === 'STRONG') {
+      size = 'wide';
+      if (el.firstElementChild.firstElementChild?.tagName === 'U') {
+        size = 'super-wide';
+      }
+    }
+    return [el.textContent?.trim(), size];
+  });
+}
+
+async function initMerchCards(config, type, filtered, el, preferences) {
+  let cardsData;
+  let err;
+
+  try {
+    const res = await fetch(`${config?.locale?.prefix ?? ''}${config.queryIndexCardPath}.json?sheet=${type}`);
+    if (res.ok) {
+      cardsData = await res.json();
+    } else {
+      err = res.statusText || res.status;
+    }
+  } catch (error) {
+    err = error.message;
+  }
+  if (!cardsData) {
+    fail(el, err);
+  }
+
+  // TODO add aditional parameters.
```

Comment:
Is this still relevant and an outstanding task? If yes, is there a follow-up ticket for this?

Reply:
I think it is not relevant, removing it.